### ISSUE TITLE
Audio Encoding: specified bitrate should only down-code and not up-code

### ISSUE THREAD

**Issue by Bardmaster2** [RandomEngy/VidCoder#919]
I would love to see an audio encoding option to down-code bitrate to x value or keep the source value if lower, similar to the video Sizing where you can select "downscale only" to downscale to a certain width/height or just pass through the source video's resolution if less than that.  Alternatively just default this behavior when a Bitrate value is specified.

Currently, if I set the Bitrate to a specific value, VidCoder is up-coding lower bitrates to that value, which is just a waste of space.

My use-case:  I am transcoding a large media library to H.265 to save space.  Part of my approach is to convert everything to AAC where the bitrate should be no more than 256.  I want to keep the source bitrate if <256 and down-code if >256.

Thank you for listening!

**Comment by RandomEngy** [comment]
Are you asking for something like conditional passthrough? Like, if the bitrate is below this value, pass it through, but if it's above the given bitrate, re-encode?

Or are you asking for something that is always re-encoding but picking the bitrate differently based on the input?

**Comment by Bardmaster2** [comment]
The first one, conditional passthrough.

**Comment by RandomEngy** [comment]
Makes sense and sounds like a handy feature. I'll put it on the backlog. Though it might be a little tricky to implement since I'll need to deal with quality targeting, bitrate detection on different input codecs, and settings on the fallback encoder.

**Comment by Bardmaster2** [comment]
Hi @RandomEngy any update on this enhancement?  Thanks & great work overall, VidCoder is my go-to transcoding tool!

**Comment by RandomEngy** [comment]
I haven't worked on it; been busy with other projects.
